src/payment/checkout/payment-form/StripePaymentForm.jsx

```diff
@@ -241,37 +202,24 @@ function StripePaymentForm({
 }
 
 StripePaymentForm.propTypes = {
-  disabled: PropTypes.bool,
-  enableStripePaymentProcessor: PropTypes.bool,
   handleSubmit: PropTypes.func.isRequired,
   isBulkOrder: PropTypes.bool,
-  loading: PropTypes.bool,
   isQuantityUpdating: PropTypes.bool,
   isProcessing: PropTypes.bool,
   onSubmitButtonClick: PropTypes.func.isRequired,
+  onSubmitPayment: PropTypes.func.isRequired,
   options: PropTypes.object, // eslint-disable-line react/forbid-prop-types,
   submitErrors: PropTypes.objectOf(PropTypes.string),
-  products: PropTypes.array, // eslint-disable-line react/forbid-prop-types,
-  issueError: PropTypes.func.isRequired,
-  skuError: PropTypes.func.isRequired,
 };
 
 StripePaymentForm.defaultProps = {
-  disabled: false,
-  enableStripePaymentProcessor: true,
   isBulkOrder: false,
-  loading: false,
   isQuantityUpdating: false,
   isProcessing: false,
   submitErrors: {},
-  products: [],
   options: null,
 };
 
 export default reduxForm({ form: 'stripe' })(connect(
   null,
```

Comment:
Is the connect still needed if all that is passed is 'null'?

Reply:
Why wouldn't [`connect()`](https://react-redux.js.org/api/connect) be needed if all that is passed is null?